solr/config/solrconfig.xml

```diff
@@ -1180,7 +1180,6 @@
       <str>yyyy-MM-dd HH:mmZZ</str>
 
       <str>yyyy-MM-dd</str>
-      <str>yyyy</str>
```

Comment:
This was just added as of #25 .
Why remove it?

Reply:
Shouldn't of been added. The date object requires month and day. It was effectively doing nothing until trying to index a pdate with year only.